src/Reflection/StringCast/ReflectionPropertyStringCast.php

```diff
@@ -37,6 +39,18 @@ public static function toString(ReflectionProperty $propertyReflection): string
         );
     }
 
+    /** @psalm-pure */
+    private static function docCommentToString(ReflectionProperty $propertyReflection, bool $indent): string
+    {
+        $docComment = $propertyReflection->getDocComment();
+
+        if ($docComment === null) {
+            return '';
+        }
+
+        return ($indent ? preg_replace('/(\n)(?!\n)/', '\1    ', $docComment) : $docComment) . "\n";
+    }
```

Comment:
This doc comment indent logic is repeated multiple times already: perhaps worth externalizing?

The logic is roughly:

* give an indent number
* apply indent if the docComment is not empty

Reply:
Added `ReflectionStringCastHelper`. However I've preserved the boolean parameter but changed the name to be more meaningful.